oggm_3dviz/tools/viz.py

```diff
@@ -550,29 +550,78 @@ def update_glacier(self, step, camera_position_per_step=None):
         self.plotter.update()
 
     def export_animation(self, filename="animation.mp4", framerate=10,
-                         quality=5, moving_camera_start_and_end_point=None,
+                         quality=5, camera_trajectory='linear', linear_camera_start_and_end_point=None,
```

Comment:
- Can you set as default a non moving camera `camera_trajectory=None`.
- Instead of having kwargs of the different options for the moving camera you can use `kwarg_camera_trajectory` (so for linear it should contain `kwarg_camera_trajectory['start_point']` and `kwarg_camera_trajectory['end_point']` ). The user needs to make sure that all needed arguments are provided.

Reply:
yes good point